libraries/rush-lib/src/api/RushConfiguration.ts

```diff
@@ -1518,4 +1575,14 @@ export class RushConfiguration {
     }
     return undefined;
   }
+
+  public async _loadCurrentVariantJsonAsync(): Promise<ICurrentVariantJson | undefined> {
+    try {
```

Comment:
This file access would be unnecessary if we only populated the `currentVariantJsonFilePath` when at least one other variant is possible

Reply:
True, but if someone eliminated all secondary variants and then ran `rush install`, we may erroneously skip the install.